templates/e-commerce/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
```

Comment:
That makes sense - can you work on a name for this template as well as this component with @spacedino - this is not going to be merge blocking